Assets/PlayerControl.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerControl : MonoBehaviour {
+    public float moveSpeed;
+    public Rigidbody2D rb;
```

Comment:
```suggestion
[SerializeField] private float moveSpeed;
[SerializeField] private Rigidbody2D rb;
```

probably dont wanna make these public
SerializeField exposes the variables to Unity inspector but not to other classes so its good